test

```diff
@@ -88,6 +94,60 @@ function integration_pass {
 	go test -timeout 1m -v ${RACE} -cpu 1,2,4 -run=Example $@ ${TEST}
 }
 
+function functional_pass {
+	rm -rf ./agent-*
+
+	mkdir -p ./agent-1
+	./bin/etcd-agent -etcd-path ./bin/etcd -etcd-log-dir ./agent-1 -port :19027 -use-root=false &
+	AGENT_PID_1=$!
+	echo "AGENT_PID_1:" ${AGENT_PID_1}
+
+	mkdir -p ./agent-2
+	./bin/etcd-agent -etcd-path ./bin/etcd -etcd-log-dir ./agent-2 -port :29027 -use-root=false &
+	AGENT_PID_2=$!
+	echo "AGENT_PID_2:" ${AGENT_PID_2}
+
+	mkdir -p ./agent-3
+	./bin/etcd-agent -etcd-path ./bin/etcd -etcd-log-dir ./agent-3 -port :39027 -use-root=false &
+	AGENT_PID_3=$!
+	echo "AGENT_PID_3:" ${AGENT_PID_3}
+
+	# kill members, trigger snapshot
+	sleep 3s
+	./bin/etcd-tester \
+		-agent-endpoints "127.0.0.1:19027,127.0.0.1:29027,127.0.0.1:39027" \
+		-client-ports 12379,22379,32379 \
+		-peer-ports 12380,22380,32380 \
+		-limit 1 \
+		-schedule-cases "0 1 2 3 4 5" \
+		-exit-on-failure &
```

Comment:
why does this need to be `&`? can't it launch then exit normally without dealing with `wait`?